sparse_ho/implicit_forward.py

```diff
@@ -93,20 +84,9 @@ def get_only_jac(
                 Xs, y, r, dbeta, dr, L, alpha, sign_beta)
 
         objs.append(
-            model.get_jac_obj(Xs, y, sign_beta, dbeta, r, dr, alpha))
-        # m1 = norm(- v.T @ Xs.T @ dr + sign_beta * n_samples * alpha)
-        # m2 = tol_jac * np.sqrt(n_features) * n_samples * alpha * norm(v)
-        # crit = m1 <= m2
-        # print("m1 %.2f", m1)
-        # print("m2 %.2f", m2)
-        # print("m1 = %f" % norm(v @ (dbeta - dbeta_old)))
-        # print("tol_crit %f" % tol_crit)
-        # if norm(v @ (dbeta - dbeta_old)) < tol_crit:
-        # if norm((dbeta - dbeta_old)) < tol_jac * norm(dbeta):
-        # crit =
-        # print('jac obj', objs[-1])
```

Comment:
cleanup?

Reply:
This is very werid, my ```implicit_forward.py``` does not have more than 100lines:
https://github.com/QB3/sparse-ho/blob/1313e032f28b7e0f85649367b51dbbe829249813/sparse_ho/implicit_forward.py